src/pages/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import { GET_PRODUCTS_API, POST_ADD_CART_API } from '../../config.js';
+import { FaCreativeCommons, FaCreativeCommonsPdAlt } from 'react-icons/fa';
+import { RiArrowDownSLine, RiArrowUpSLine } from 'react-icons/ri';
+
+import './ProductDetail.scss';
+
+export class ProductDetail extends Component {
+  constructor() {
+    super();
+    this.set = React.createRef();
+    this.state = {
+      productData: {},
+      recommendToggleOn: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${GET_PRODUCTS_API}/${this.props.match.params.productID}`, {
+      headers: { Authorization: localStorage.getItem('access_token') },
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productData: data.message[0],
+        });
+      });
+  }
+
+  postCart = () => {
+    const id = this.props.match.params.productID;
+    if (this.state.productData.cart_exist) {
+      return;
+    } else {
+      fetch(`${POST_ADD_CART_API}`, {
+        method: 'POST',
+        body: JSON.stringify({
+          productID: id,
```

Comment:
```suggestion
          productID: this.props.match.params.productID;
```
여기서 바로 사용해도 무방해보입니다!